guacamole/src/main/webapp/app/settings/services/preferenceService.js

```diff
@@ -98,6 +98,17 @@ angular.module('settings').provider('preferenceService', ['$injector',
         return language.replace(/-/g, '_');
 
     };
+    
+    /**
+     * Return the timezone detected for the current browser session
+     * by the JSTZ timezone library.
+     * 
+     * @returns String
+     *     The name of the currently-detected timezone.
```

Comment:
Is the format of the timezone returned by JSTZ known to conform to a particular standard? If so, I suggest documenting that here.

Reply:
Sounds good.  Is there any precedent to adding a link/URL within the JavaDoc for this to the Wikipedia article on tz database?